File: backend/src/main/java/com/wgzhao/addax/admin/model/EtlJob.java
package com.wgzhao.addax.admin.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "etl_job")
@Setter
@Getter
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EtlJob
{
    @Id
    long tid;

    String job;
}

File: src/main/kotlin/com/wgzhao/addax/admin/model/EtlJob.kt
package com.wgzhao.addax.admin.model

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "etl_job")
data class EtlJob(
    @Id
    val tid: Long = 0,
    val job: String? = null
)
